### PR TITLE
Allow Multiple Whitelist IPs

### LINKED ISSUE #57: Only one whitelist IP can be defined via the .env file
This only allows 1 IP to be defined within the .env file. 

Getting the environment value grabs a whole string:
https://github.com/akaunting/laravel-firewall/blob/72f30fa7962e4682aca341ba273156dbc483da6b/src/Config/firewall.php#L7

When it's used it is only treated as containing 1 IP in a string:
https://github.com/akaunting/laravel-firewall/blob/72f30fa7962e4682aca341ba273156dbc483da6b/src/Traits/Helper.php#L24

### PR DESCRIPTION
**Pull request by carlos-reynosa**
Fixes #57 Allow multiple whitelist IPs to be able to be defined within the Whitelist IP environment variable.

### CODE REVIEW

**Comment by humaneguy** [comment]
Kindly append tests to this PR as that helps to ensure everything works smoothly as possible. Thank you. It also helps us understand better what you're trying to do.

**Comment by humaneguy** [comment]
Just figured;😅
The way whitelist works pretty much starts from [here](https://github.com/akaunting/laravel-firewall/blob/master/src/Traits/Helper.php#L22-L25). Which then utilizes this Symfony helper method:

```php
public static function checkIp($requestIp, $ips)
    {
        if (!\is_array($ips)) {
            $ips = [$ips];
        }

        $method = substr_count($requestIp, ':') > 1 ? 'checkIp6' : 'checkIp4';

        foreach ($ips as $ip) {
            if (self::$method($requestIp, $ip)) {
                return true;
            }
        }

        return false;
    }
```

So you see it actually supports:
- An array of IPs
- A single IP.

However, the default value can be:
```php
'whitelist' => [env('FIREWALL_WHITELIST', '')],
```

or

```php
'whitelist' => [env('FIREWALL_WHITELIST', [])],
```

Nothing is broken.

TLDR;

You can pass an array of values and this package will work properly. Hence, no need for this PR.

**Comment by carlos-reynosa** [comment]
I guess my expectation was that I should be able to provide a list of whitelist IPS using the provided environment variable. Currently you cannot do that unless you list them out directly withing the config file or you change the config code to parse a comma delimited list of IPs provided from the environment variable.

**Comment by carlos-reynosa** [comment]
Providing the list of IPs delimited by a ":" within the FIREWALL_WHITELIST environment variable does not work. The process does not parse out each IP and does not create an array of IPs for the function checkIp to check individually. Thus if you provide more than one IP it fails. It treats the value "127.0.0.1:192.168.0.1" as one IP when it should be treated as two IPs. 

The pull request i provided  makes it so that the environment variable can be a list of IPs delimited by some character (",") and results in an array of IPs being passed into the checkIp function. 

My expectation was that we should be able to provide multiple IPs through the  FIREWALL_WHITELIST environment variable.

**Comment by carlos-reynosa** [comment]
I added a test case that uses the value you're saying should work. It should actually fail. Maybe that might help illustrate what i'm seeing.

File: tests/Feature/WhitelistTest.php [hunk 1]
```diff
@@ -15,6 +15,13 @@ public function testShouldAllow()
         $this->assertEquals('next', (new Whitelist())->handle($this->app->request, $this->getNextClosure()));
     }
 
+    public function testShouldAllowMultiple()
+    {
+        config(['firewall.whitelist' => ['127.0.0.0/24:0.0.0.0']]);
```
Comment (carlos-reynosa) [inline tests/Feature/WhitelistTest.php:20]: This would fail because nothing after it actually parses it out into a list of IPs. It needs to be parsed out and turned into an array before being stored in the config value. This is what my pull request does.

Comment (swilla) [reply tests/Feature/WhitelistTest.php:20]: Also, we should consider that separating these by a `:` will be problematic for ipv6.
